### DISCUSSION TITLE
Resuming training after system failure

### DISCUSSION THREAD

**Discussion by planetrocke** [coqui-ai/TTS#1651] [General Q&A] [answered]
I've been having issues with either power or something else, leading to crashes. Is there any way to resume training after this type of event?

**Comment by domcross** [comment]
Try using `train_tts.py` with option `--continue_path` pointing to the config.json created in the train folder from the previous run.

**Comment by planetrocke** [comment, marked as answer]
This worked with trainer.distribute ......... --continue_path vits_folder_path (no config.json). I also noticed it takes the last known best and seems to start from there. 

```
python3 -m trainer.distribute --script recipes/ljspeech/vits_tts/train_vits.py  --continue_path <vits_folder_name> --gpus 0,1,2,3,4,5
```
